deploy/kubernetes/releases/csi-digitalocean-dev/crds.yaml

```diff
@@ -27,9 +27,8 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
```

Comment:
Copied new CRDs from https://github.com/kubernetes-csi/external-snapshotter/tree/v8.0.1/client/config/crd
Important change here is that they moved validation from the webhook to the CEL rules on the CRDs themselves, so I'm pulling in all that validation.
They've also added crds for "volumegroupsnapshots" but that's irrelevant for DO cause DO volumes don't support taking a snapshot of multiplt volumes at once